src/preamble.js

```diff
@@ -1097,12 +1102,12 @@ function createWasm() {
 #if RUNTIME_DEBUG
   dbg('asynchronously preparing wasm');
 #endif
+  instantiateAsync(wasmBinary, wasmBinaryFile, info).then(receiveInstantiationResult)
 #if MODULARIZE
   // If instantiation fails, reject the module ready promise.
-  instantiateAsync(wasmBinary, wasmBinaryFile, info, receiveInstantiationResult).catch(readyPromiseReject);
-#else
-  instantiateAsync(wasmBinary, wasmBinaryFile, info, receiveInstantiationResult);
+  .catch(readyPromiseReject);
```

Comment:
```suggestion
  .catch(readyPromiseReject)
```
The new `;` line below provides this